first-analysis-steps/split-jobs.md

```diff
@@ -28,6 +28,12 @@ and reduces the impact of jobs failing due to grid problems.
 Setting `filesPerJob=5` should work well for real data, while `filesPerJob=1` should be good for signal MC.
 {% endcallout %} 
 
+{% callout Splitter arguments %}
```

Comment:
I think you are missing the "quotation marks" around the "Splitter arguments". Probably that's why CI failed

Reply:
indeed, thanks